### ISSUE TITLE
Re-rendering issue

### ISSUE THREAD

**Issue by rakhadkz** [syncfusion/ej2-react-ui-components#86]
So it started to re-render the schedule components each time when mouse hovers over the context menu parent item. Also update stopped updating status and technicians from context menu

https://www.loom.com/share/ec7c66c82e9342d9bab45eda4807c848

Versions

```
    "@syncfusion/ej2-base": "^19.1.64",
    "@syncfusion/ej2-data": "^19.1.63",
    "@syncfusion/ej2-react-buttons": "^19.1.63",
    "@syncfusion/ej2-react-calendars": "^19.1.64",
    "@syncfusion/ej2-react-dropdowns": "^19.1.65",
    "@syncfusion/ej2-react-navigations": "^19.1.63",
    "@syncfusion/ej2-react-schedule": "^19.1.65",
```

**Comment by tonydehnke** [comment]
Can confirm issue remains in 19.1.66 too.

**Comment by tonydehnke** [comment]
Issue remains in 19.1.67

**Comment by Hareesh22** [comment]
Dear Customer,

Greetings from Syncfusion Support..!
We have prepared a sample to reproduce your reported issue “Scheduler re-renders while we navigate through the context menu options” unfortunately, we were unable to reproduce your reported problem and the sample can be viewed from the following link. We have tried the following ways to replicate the issue at our end.

1.	We have ensured your reported problem by preparing a default sample with context menu options and in that, we have ensured by moving the cursor through the context menu options but the scheduler is not refreshed at our end.
2.	Also, we have ensured the below sample in a different browser like Chrome, Edge and it is rendering properly in it.
3.	Ensured your reported problem in the reported version 19.1.67v.

**Sample**: https://stackblitz.com/edit/react-schedule-with-context-menu-sample?file=index.js
**UG link**: https://ej2.syncfusion.com/react/documentation/schedule/context-menu/

Kindly refer to the above sample, also share the below details to validate your reported issue and serve you better.
•	Confirm whether our shared sample is working properly or not at your end?
•	Whether you are done any customization for the Scheduler/context menu?
•	Share the entire scheduler rendering code snippets.
•	Is the issue occurs in a specific case or specific browser or specific device?
•	Replicate the issue in the above sample.
•	Share issue depicting video demo
•	Share the issue replicating sample (if possible)
.
We will be happy to assist you..!

Regards,
Hareesh
